### PR TITLE
Clips redesign

### PR DESCRIPTION
**Pull request by mitchdowney**

### CODE REVIEW

File: Podverse/ClipsListContainerViewController.swift [hunk 1]
```diff
@@ -142,30 +166,60 @@ extension ClipsListContainerViewController:UITableViewDelegate, UITableViewDataS
...
+        if let time = time {
+            cell.time?.text = time
         }
         
+        if let episodePubDate = clip.episodePubDate {
+            cell.episodePubDate?.text = episodePubDate.toShortFormatString()
+        }
+        
+        Podcast.retrievePodcastImage(podcastImageURLString: clip.podcastImageUrl) { (podcastImage) -> Void in
+            DispatchQueue.main.async {
```
Comment (kreonjr) [inline Podverse/ClipsListContainerViewController.swift]: I believe the completion is already being returned on the main queue and if it isn't, it should :)

File: Podverse/ClipsTableViewController.swift [hunk 2]
```diff
@@ -125,18 +131,14 @@ extension ClipsTableViewController:UITableViewDelegate, UITableViewDataSource {
             cell.episodePubDate?.text = episodePubDate.toShortFormatString()
         }
         
-        DispatchQueue.global().async {
-            var cellImage:UIImage?
-            // TODO: remotely retrieve cell image, if it isn't saved with a podcast locally
-            cellImage = UIImage(named: "PodverseIcon")
-
+        Podcast.retrievePodcastImage(podcastImageURLString: clip.podcastImageUrl) { (podcastImage) -> Void in
             DispatchQueue.main.async {
```
Comment (kreonjr) [inline Podverse/ClipsTableViewController.swift]: Same comment about main queue as above

File: Podverse/Podcast.swift [hunk 3]
```diff
@@ -69,7 +69,9 @@ class Podcast: NSManagedObject {
         }
         else if let imageUrlString = podcastImageURLString, let imageURL = URL(string:imageUrlString) {
             Podcast.fetchPodcastImage(podcastImageUrl: imageURL, completion: { (image) in
-                completion?(image)
+                DispatchQueue.main.async {
+                    completion?(image)
+                }
```
Comment (mitchdowney) [inline Podverse/Podcast.swift:74]: Like this?

Comment (kreonjr) [reply Podverse/Podcast.swift:74]: Yup exactly. And doing so, the callers don't need to dispatch to the main queue themselves :)

**Comment by kreonjr** [comment]
Please resolve merge conflict and it's good to go

**Pull request merged**